.github/workflows/autoupdate-pre-commit-config.yml

```diff
@@ -17,10 +17,16 @@ jobs:
     steps:
       - name: Set up Python
         uses: actions/setup-python@v2
-      - uses: actions/cache@v2
+      - name: Cache multiple paths
```

Comment:
the pre-commit action already does caching, are you sure we need this?

Reply:
From the pre-commit documentation, it looks like we have to explicitly have the pre-commit cache. https://pre-commit.com/#github-actions-example. Also no where in the doc I found pre-commit is caching the `pre-commit cache` directory.

Reply:
> the pre-commit action already does caching, are you sure we need this?

I also thought that pre-commit caches.

Reply:
This is for autoupdating pre-commit, which requires manual caching of `~/.cache/pre-commit` for a lil speedup. 
Having `os.runner` makes no sense  since it only runs on linux and `env.cache-name` isn't defined, but well it will be an empty string.
What takes care of caching itself is `pre-commit/action@v2.0.0`
https://github.com/pre-commit/action/blob/bedfaa0b2473eb8caef27882abfbc43492269c6b/index.js#L54

Reply:
Oh ok, then pre-commit v2 caches. I was looking at the documentation and I saw the Caching section for every CI (github, gitlab) and I concluded that we would need manual caching.

Reply:
Ok so we don't need to cache this action - I will remove it from the pre-commit gh action, if that works for us.

Reply:
* For this file `autoupdate-pre-commit-config.yml`, we do need to cache only `~/.cache/pre-commit`. 
* And since this action installs `pre-commit`, then we can cache `~/.cache/pip` on linux and `~\AppData\Local\pip\Cache` on windows. 
* We can remove rest of the paths which are `c:\hostedtoolcache\windows\python\*` and `/opt/hostedtoolcache/Python/*`. And we dont need to cache these directories since these are the hosted tool cache directories and they could change with every runner build.